game/menu.hh

```diff
@@ -29,7 +29,12 @@ public:
 	MenuOption(const std::string& nm, const std::string& comm, MenuImage img = MenuImage());
 
 	/// Make the option change values of a ConfigItem.
-	MenuOption& changer(ConfigItem& val) { type = CHANGE_VALUE; value = &val; return *this; }
+	MenuOption& changer(ConfigItem& val, std::string virtOptName = std::string()) {
+		type = CHANGE_VALUE;
+		value = &val;
+		if (!virtOptName.empty()) virtualName = virtOptName;
```

Comment:
brackets here please { }
so `if (!virtOptName.empty()) { virtualName = virtOptName; }`

this avoids any confusion